test/app/digest.result

```diff
@@ -636,3 +629,173 @@ res
   - bafac115a0022b2894f2983b5b5102455bdd3ba7cfbeb09f219a9fde8f3ee6a9
   - bafac115a0022b2894f2983b5b5102455bdd3ba7cfbeb09f219a9fde8f3ee6a9
 ...
+--
+-- gh-2003 xxHash
+--
+xxhash32 = digest.xxhash32.new()
+---
+...
+xxhash32
+---
+- 'value: 46947589'
```

Comment:
Why this value? Isn't it random? Because I don't see where the state is initialized, so the state must be garbage until you call `clear()`. Which also might be an issue. Probably you must call `clear()` internally always inside of `new()`.

Reply:
ffi.new fills structure with zero bytes. It seems to be predictable. However, it's reasonable to choose some default seed as e.g. for murmur.
```
tarantool> require('digest').murmur
---
- new: 'function: 0x010ea89a90'
  default_seed: 13
...
```

Reply:
I made seed as optional value and by default it's 0 (https://github.com/tarantool/tarantool/pull/5828/commits/776c325334459c4b867537a417043062b8a5e874)